applications/ContactStructuralMechanicsApplication/custom_python/add_custom_utilities_to_python.cpp

```diff
@@ -39,9 +37,7 @@ void  AddCustomUtilitiesToPython()
     class_<TreeContactSearch<2, 2>>("TreeContactSearch2D2N", init<ModelPart&>())
     .def(init<ModelPart&, Parameters>())
     .def("InitializeMortarConditions",&TreeContactSearch<2, 2>::InitializeMortarConditions)
-    .def("ClearScalarMortarConditions",&TreeContactSearch<2, 2>::ClearScalarMortarConditions)
```

Comment:
If this is no longer used, should we remove its code as well or is there any reason to keep it?
https://github.com/KratosMultiphysics/Kratos/blob/03bb8c72655d5f7acba34eeb35385b3c82c24759/applications/ContactStructuralMechanicsApplication/custom_utilities/tree_contact_search.h#L279

Same with the others

Reply:
I already removed, this is a residual of the conflict of multiple merges